src/lib.c

```diff
@@ -387,7 +428,7 @@ efi_save_esp_filename(void)
 
 	search_esp_filename(esp_filename, esp_filename_path);
 	if (esp_filename_path[0] != '\0') {
-		printf("efivar: VarToFile content saved at %s\n", esp_filename_path);
```

Comment:
The print should be removed from the original patch

Reply:
Done